### PR TITLE
Random port

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: share/postgresql-setup/postgresql_pkg_tests.sh.in [hunk 1]
```diff
@@ -4,11 +4,23 @@
 
 # This file is to be sourced.
 
+try_random_port ()
+{
+    _port=$RANDOM
+    if test -n "$_port" && eval '_port=$(( $_port + 32000 ))' 2>/dev/null; then
+        eval 'echo "$_port"'
```
Comment (pkubatrh) [inline share/postgresql-setup/postgresql_pkg_tests.sh.in]: I am wondering - why `eval echo` instead of just `echo`?

**Comment by pkubatrh** [comment]
otherwise lgtm

File: share/postgresql-setup/postgresql_pkg_tests.sh.in (hunk 1 above)
Comment (praiskup) [reply share/postgresql-setup/postgresql_pkg_tests.sh.in]: Good question :-) I tried to silence ShellCheck warning (used but unset _port variable).  But I can avoid using that...

**Comment by praiskup** [comment]
I'll drop the eval, and merge.  Thanks for the review!

**Pull request merged**